bytestring.cabal

```diff
@@ -135,7 +135,11 @@ library
   -- DNDEBUG disables asserts in cbits/
   cc-options:        -std=c11 -DNDEBUG=1
  
-  if os(windows)
+  -- Required, due to the following issues:
+  -- * https://gitlab.haskell.org/ghc/ghc/-/issues/20525#note_385580
+  -- * https://gitlab.haskell.org/ghc/ghc/-/issues/19417
```

Comment:
This is outdated, the comment was removed in 0066cf8.

Reply:
Alright, I'll drop it.